documentation/library_guide/parallel_api/execution_policies.rst

```diff
@@ -117,17 +117,17 @@ and ``using namespace sycl;`` directives when referring to policy classes and fu
 
 .. code:: cpp
 
-   auto policy_a = device_policy<class PolicyA> {};
+   auto policy_a = device_policy<class PolicyA>();
    std::for_each(policy_a, ...);
 
 .. code:: cpp
 
-  auto policy_b = device_policy<class PolicyB> {device{gpu_selector{}}};
+  auto policy_b = device_policy<class PolicyB>(device(gpu_selector_v));
   std::for_each(policy_b, ...);
 
 .. code:: cpp
 
-  auto policy_c = device_policy<class PolicyС> {cpu_selector{}};
```

Comment:
Incorrect codepage of lerrer ```С``` in ```PolicyС```; C++ compile error

Reply:
```cpu_selector``` is deprecated: https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#sec:device-selector